server/utils/formUtils.ts

```diff
@@ -18,4 +18,12 @@ export default class FormUtils {
 
     res.locals.errors = { list, messages }
   }
+
+  static setFormValues(req: Request, res: Response): void {
```

Comment:
I was thinking we must have had this need before, but I don't think we've had any form pages where there could be both invalid and truthy data 😱